go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	golang.org/x/crypto v0.0.0-20190129210102-ccddf3741a0c
 	golang.org/x/net v0.0.0-20180906233101-161cd47e91fd
 	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f
-	golang.org/x/sys v0.0.0-20180909124046-d0be0721c37e
+	golang.org/x/sys v0.0.0-20190321052220-f7bb7a8bee54
```

Comment:
@AkihiroSuda Did you update this manually? It doesn't seem to match what is in containerd repo

Reply:
from `go get github.com/golang.org/x/sys@master`

Reply:
Could we use the containerd commit instead to avoid any accidental conflicts?